src/web/app/pages-admin/admin-home-page/admin-home-page.component.ts

```diff
@@ -236,6 +247,45 @@ export class AdminHomePageComponent {
     );
   }
 
+  private formatTimestampAsString(timestamp: number, timezone: string): string {
+      const dateFormatWithZoneInfo: string = 'ddd, DD MMM YYYY, hh:mm A Z';
+
+      return this.timezoneService
+          .formatToString(timestamp, timezone, dateFormatWithZoneInfo);
+  }
+
+  private formatAccountRequests(requests: AccountRequests): AccountRequestData[] {
+    const timezone: string = this.timezoneService.guessTimezone() || 'UTC';
+    return requests.accountRequests.map((request) => {
+      const [institute, country] = request.institute.split(', ').length === 2
+      ? request.institute.split(', ') : [request.institute, ''];
```

Comment:
I'm not sure how well this would work. Let's say `request.institute` is `` `${instituteAbc}, ${countryXyz}` ``. Suppose `instituteAbc` had commas in it, then the length after doing `split` would be more than two, so the value assigned to `[institute, country]` would be ``[`${instituteAbc}, ${countryXyz}`, '']``, so `countryXyz` would not be in the correct column. This should be similar for the case where `countryXyz` had commas in it.

Reply:
Thanks for the feedback, I'm also not sure whats an easy way to resolve this, but I changed it to split at the last comma instead of  the first, so this at least guarantees a length of 2, and also since it is more likely that the institute contains a comma, as compared to the country. Otherwise, do you have any suggestions on how to handle this?

Reply:
For me, I feel like it's easier to just keep it as one institute column, instead of splitting it into two columns, but I'm not sure what's the best option either.

@ziqing26, @EuniceSim142, do you have any thoughts on this?

Reply:
Here's the current code for reference 
```
const lastCommaIndex = request.institute.lastIndexOf(', ');
const [institute, country] = lastCommaIndex === -1
        ? [request.institute, '']
        : [request.institute.slice(0, lastCommaIndex), request.institute.slice(lastCommaIndex + 2)];
```

Reply:
I feel like what you suggested where you split at the last comma may not be too bad actually. You're right that a comma in a country's name is rare.

It seems like there is currently only one country with a comma in their name on [this list](https://worldpopulationreview.com/country-rankings/official-names-of-countries), and it's in their official name, "Nation of Brunei, Abode of Peace", which I doubt anybody is likely to use for TEAMMATES. Besides, it seems as though it should be "Brunei Darussalam" instead? Also, after skimming [the list on Wikipedia](https://en.wikipedia.org/wiki/List_of_sovereign_states), I could not even find any countries with commas in their names.

So, yea, we can probably settle with what you suggested, unless the others have anything to say about it. Just hope that no new country appears with a comma in its name, or no existing country renames itself. 😁

With this, I wonder if we can just migrate towards having an actual country column in the database. I doubt there are many cases with multiple commas to begin with, so those can be corrected manually. But anyway, this is out of the scope of this PR.

Reply:
Let's keep it to be displayed as one column in the table i.e. "Institute, Country"
Because (1) currently the db's institute field contain both institute and country (i.e. it is a string editable by admin) (2) we are not using dropdown in the frontend for this field due to sensitivity of region v.s. country in some scenarios, so no guarantee on the format of country string

Reply:
Updated with the above-mentioned changes, so the institute and country variables are now combined as `instituteAndCountry` on the `AccountRequestTableRowModel`